node/src/components/consensus/protocols/zug.rs

```diff
@@ -199,6 +201,8 @@ where
     next_scheduled_update: Timestamp,
     /// The write-ahead log to prevent honest nodes from double-signing upon restart.
     write_wal: Option<WriteWal<C>>,
+    /// The list of the node from which we wait for a sync response.
+    expected_to_send_a_sync_response: registered_sync::RegisteredSync,
```

Comment:
It's keyed by `RandomId`, so it's not a list of nodes, is it? Maybe better to call it something like `sent_sync_requests`, and comment that it is a map from the request ID to the timestamp when it was sent.

Reply:
You might have forgotten to change the comment :wink:

Reply:
Sorry…